projects/jump_ci/testing/test.py

```
@@ -0,0 +1,254 @@
+configure_logging()
+
+from projects.jump_ci.testing import utils, prepare_jump_ci, tunnelling
+
+def rewrite_variables_overrides(variable_overrides_dict, nb_args_to_eat):
+    new_variable_overrides = dict()
+
+    old_args = (variable_overrides_dict["PR_POSITIONAL_ARGS"] or "").split()
+    # remove the first args of the jump-ci test (project and cluster)
+    new_args = old_args[nb_args_to_eat:]
+
+    new_args_str = new_variable_overrides[f"PR_POSITIONAL_ARGS"] = " ".join(new_args)
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)